src/actions/__snapshots__/sequencerInbox.integration.test.ts.snap

```diff
@@ -0,0 +1,10 @@
+// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
+
+exports[`max time variation management > getMaxTimeVariation successfully fetches max time variation 1`] = `
```

Comment:
i don't think it's worth adding a snapshot for a small test case like this, let's just inline